java/cdk/src/main/java/sleeper/cdk/stack/GarbageCollectorStack.java

```diff
@@ -87,7 +89,7 @@ public GarbageCollectorStack(
                 .runtime(Runtime.JAVA_11)
                 .handler("sleeper.garbagecollector.GarbageCollectorTriggerLambda::handleRequest")
                 .environment(Utils.createDefaultEnvironment(instanceProperties))
-                .reservedConcurrentExecutions(1)
+                .reservedConcurrentExecutions(instanceProperties.getInt(GARBAGE_COLLECTOR_LAMBDA_CONCURRENCY_RESERVED))
```

Comment:
This should be set on the handler function. The trigger function needs this set to 1.

Reply:
Address